src/dev/build/tasks/patch_native_modules_task.ts

```diff
@@ -52,31 +52,42 @@ interface Package {
   >;
 }
 
+// Process for updating urls and checksums after bumping the version of `re2`:
+// 1. Match `version` with the version in the yarn.lock file.
+// 2. Update the url to match the version.
+//    2a. If a Node.js update occurs, the node module version must match as
+//        well (i.e. '83'). See https://nodejs.org/en/download/releases/#ref-1.
+// 3. Generate the new checksum by executing the following commands:
+//    3a. `wget {url}`
+//    3b. `sha256sum {downloaded file name}`
+//    3c. For `linux-arm64`, the sha256 can also be found by replacing
+//        "linux-arm64-83.tar.gz" in the url with "sha256sum.txt.asc"
+//        and copying the sha256 from that file.
```

Comment:
Thanks for adding these steps @tmarkley. Tangential question, why do we need this patching process?

Reply:
Good question. This patch existed before the fork, but I know @ananzh did the work to bring the ARM build of re2 in. I haven't done a ton of digging in to why this exists, but I'd love to find an alternative approach.

Reply:
Installing re2 produces a binary for the specific platform architecture it was installed on. However, `yarn build-platform` can be used to create distributables for platforms other than the one the command is being executed on and for this, it downloads NodeJS executable and any binaries needed for the platforms.